OpenRA.Mods.Common/Projectiles/NukeLaunch.cs

```diff
@@ -64,7 +64,7 @@ public class NukeLaunch : IProjectile, ISpatiallyPartitionable
 				Game.Sound.Play(SoundType.World, weapon.Report.Random(firedBy.World.SharedRandom), pos);
 
 			if (skipAscent)
-				ticks = turn;
```

Comment:
Changing this but not the offset above makes the missiles move visually at half the speed when skipping the ascent.

You can fix and simplify this by removing this if statement and changing the code above to look like:
```csharp
turn = skipAscent ? 0 : delay / 2;
var offset = new WVec(WDist.Zero, WDist.Zero, velocity * (delay - turn));
```
and then also
```csharp
pos = skipAscent ? descendSource : ascendSource;
```
otherwise the projectile will move at half speed.